src/main/java/org/jabref/gui/externalfiles/AutoLinkFilesAction.java

```diff
@@ -66,14 +66,19 @@ public void execute() {
 
             @Override
             protected void succeeded() {
-                if (!getValue().isEmpty()) {
-                    if (nc.hasEdits()) {
-                        nc.end();
-                        undoManager.addEdit(nc);
+                if (util.returnFileExceptions().isEmpty()) { //change
+                    if (!getValue().isEmpty()) {
+                        if (nc.hasEdits()) {
+                            nc.end();
+                            undoManager.addEdit(nc);
+                        }
+                        dialogService.notify(Localization.lang("Finished automatically setting external links."));
+                    } else {
+                        dialogService.notify(Localization.lang("Finished automatically setting external links.") + " " + Localization.lang("No files found."));
                     }
-                    dialogService.notify(Localization.lang("Finished automatically setting external links."));
                 } else {
-                    dialogService.notify(Localization.lang("Finished automatically setting external links.") + " " + Localization.lang("No files found."));
+
+                    dialogService.notify(Localization.lang("Problem finding files")); //change
```

Comment:
You need to add another else case when the exceptions are not empty:
`Problem finding files.  Errors occured See log for details`

Reply:
And when no exception and getValue is empty => No files found